packages/functions/src/checks/slotRelatedDocs.ts

```diff
@@ -178,12 +178,13 @@ export const findSlotBookingsMismatches = async (
       };
     }),
   }));
-  const strayBookings = normalisedEntries._reduce(collectStrayBookings, {});
+  const strayBookings = normalisedEntries
+    .filter((entry) => Boolean(Object.values(entry).length))
```

Comment:
Shouldn't the filtering happen after the transformation from `normalizedEntries` to stray bookings (operated by `collectStrayBookings`)? I haven't tried this code, so I might be wrong, but my impression is that there will be no empty object in `normalizedEntries`, so the filtering will be a no-op.
All objects in `normalizedEntries` **MUST** include an `id` AFAICT.

Reply:
You're right. I think I forgot to remove these at some point but the desired effect is achieved by the conditions ` if (Object.keys().length > 0)` inside `collectStrayBookings` and `collectInvalidIntervalBookings`.